azure-spring-cloud-maven-common/src/test/resources/pom-1.xml

```diff
@@ -0,0 +1,43 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0"
```

Comment:
Why we move the resources to `spring-cloud-maven-common` ?

Reply:
Done